src/client/cli/cmd/launch.cpp

```diff
@@ -47,15 +49,27 @@ const std::regex show{"s|show", std::regex::icase | std::regex::optimize};
 
 mp::ReturnCode cmd::Launch::run(mp::ArgParser* parser)
 {
-    auto ret = parse_args(parser);
-    if (ret != ParseCode::Ok)
+    petenv_name = Settings::instance().get(petenv_key);
+    if (auto ret = parse_args(parser); ret != ParseCode::Ok)
     {
         return parser->returnCodeFrom(ret);
     }
 
     request.set_time_zone(QTimeZone::systemTimeZoneId().toStdString());
 
-    return request_launch();
+    auto ret = request_launch();
+    if (ret == ReturnCode::Ok && request.instance_name() == petenv_name.toStdString())
+    {
+        const auto mount_source = QDir::toNativeSeparators(QDir::homePath());
+        const auto mount_target = QString{"%1:%2"}.arg(petenv_name, mp::home_automount_dir);
+        cout << fmt::format("Now auto-mounting '{}' in '{}'\n", mount_source, mount_target);
+
+        ret = run_cmd({"multipass", "mount", mount_source, mount_target}, parser, cout, cerr);
+        if (ret == ReturnCode::Ok)
+            cout << "Auto-mount succeeded\n";
```

Comment:
I don't think we should print anything on success. And failure would have printed the problem already?

Reply:
I initially did not have this, but then thought it looked strange to leave the last message on the screen saying we were mounting, while the previous one said "Launched". OTH, If I just let the mount spinner overwrite the "now mounting" message, it would not be seen (I tried this first, it would quickly be replaced with the "enabling mount support" msg). I also considered saying "Launched primary. Mounting..." in the same line so that we'd have only two lines when saying "mounted". WDYT?

Reply:
@ricab or maybe we only print on success?

Remove the above and then:
```cpp
        if (ret == ReturnCode::Ok)
            cout << fmt::format("Mounted '{}' into '{}'\n", mount_source, mount_target);
```

We still get feedback about "Enabling support for mounting…" etc, so maybe that's all that's needed?

Reply:
"Enabling…" will be almost non-existent when we go for the `sshfs` snap, so that would result in:
```
Launched: primary
Mounted '/home/michal' in 'primary:Home'
```
WDYT?

Reply:
Yep, sounds good to me. The spinner erases the "enabling..." and later msgs, so that example is also what it will look like now after it is finished.